src/utils/searchUtils.ts

```diff
@@ -149,3 +150,17 @@ export function tagsToObject(tags: Array<{ name: string; value: string }>) {
 
   return newTags;
 }
+
+export async function getAntConfirmations(id: ArweaveTransactionId) {
+  try {
+    const confirmations = await arweave.api
+      .get(`/tx/${id}/status`)
+      .then((res: any) => {
+        return res.data.number_of_confirmations;
+      });
+    return confirmations;
+  } catch (error) {
+    console.error(error);
+    return 0;
+  }
+}
```

Comment:
let's move this to `SmartweaveContractSource` interface instead of a util

Reply:
moved to interface, also had to update interface to have an arweave attribute.